dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -274,12 +274,21 @@ object Datadog {
 
     internal const val MESSAGE_ALREADY_INITIALIZED =
         "The Datadog library has already been initialized."
-    internal const val MESSAGE_NOT_INITIALIZED = "Datadog has not been initialized.\n" +
-            "Please add the following code in your application's onCreate() method:\n" +
-            "Datadog.initialize(context, \"<CLIENT_TOKEN>\");"
+    internal const val MESSAGE_NOT_INITIALIZED = "Datadog has not been " +
+        "correctly initialized.\n" +
+        "Please add the following code in your application's onCreate() method:\n" +
+        "val config = DatadogConfig" +
+        ".Builder(\"<CLIENT_TOKEN>\", \"<ENVIRONMENT>\").build()\n" +
+        "Datadog.initialize(context, config);"
+    internal const val MESSAGE_NOT_INITIALIZED_WITH_APP_ID = "Datadog has not been " +
+        "correctly initialized.\n" +
+        "Please add the following code in your application's onCreate() method:\n" +
+        "val config = DatadogConfig" +
+        ".Builder(\"<CLIENT_TOKEN>\", \"<ENVIRONMENT>\", \"<APPLICATION_ID>\").build()\n" +
+        "Datadog.initialize(context, config);"
```

Comment:
The constant should be `MESSAGE_RUMCONFIG_WITHOUT_APP_ID`. When you display it, it's when you're creating the DatadogConfig, so before the library is being initialized. I'd write something along the lines of: 
> You're trying to enable RUM but no Application Id was provided. Please use the following line to create your DatadogConfig
>   val config = DatadogConfig.Builder("<CLIENT_TOKEN>", "<ENVIRONMENT>", "<APPLICATION_ID>").build()

Reply:
Also this constant should be defined in the `DatadogConfig`'s companion object, as it's used there.